components/ProfileDropdownMenu/ProfileDropdownMenu.tsx

```diff
@@ -94,6 +98,13 @@ const ProfileDropdownMenu: React.FC<ProfileDropDownMenuProps> = ({
           >
             Profile
           </Dropdown.Item>
+          <Dropdown.Item
+            className={`${styles['dropdown-item']} `}
+            bsPrefix={isActive(userProfilePath)}
+            href={SETTINGS_ACCOUNT_PATH}
+          >
+            Setting
```

Comment:
Setting should probably be in the plural form.

Reply:
Done!